modules/Header/ui/CategoriesNav/CategoriesNavMobile.tsx

```diff
@@ -0,0 +1,114 @@
+import { translations } from '@prezly/theme-kit-nextjs/index';
+
+import { FormattedMessage } from '@/adapters/client';
+import { app } from '@/adapters/server';
+import { Dropdown } from '@/components/Dropdown';
+
+import { CategoryButton } from './CategoryButton';
+import { CategoryItem } from './CategoryItem';
+import { FeaturedCategory } from './FeaturedCategory';
+
+import styles from './CategoriesNavMobile.module.scss';
+
+export async function CategoriesNavMobile({
+    categories,
+    localeCode,
+    navigationItemClassName,
+    navigationItemButtonClassName,
+}: CategoriesNavMobile.Props) {
+    const showAllCategories = categories.length < 4;
+
+    const categoriesList = await app().categories();
+
+    function getCategory(translatedCategory: TranslatedCategory) {
+        return categoriesList.find((i) => i.id === translatedCategory.id)!;
+    }
+
+    function isCategoryFeatured(translatedCategory: TranslatedCategory) {
+        return getCategory(translatedCategory).is_featured;
+    }
+
+    const featuredCategories = categories.filter(isCategoryFeatured);
+    const regularCategories = categories.filter((i) => !isCategoryFeatured(i));
+
+    if (showAllCategories) {
+        return (
+            <>
+                {featuredCategories.length > 0 && (
+                    <li className={styles.container}>
+                        <div className={styles.featuredContainer}>
+                            {featuredCategories.map((translatedCategory) => (
+                                <FeaturedCategory
+                                    key={translatedCategory.id}
+                                    className={styles.featuredItem}
+                                    category={getCategory(translatedCategory)}
+                                    translatedCategory={translatedCategory}
+                                    size="small"
+                                />
+                            ))}
+                        </div>
+                    </li>
+                )}
+                {featuredCategories.length > 0 && regularCategories.length > 0 && (
+                    <li className={styles.container}>
+                        <hr className={styles.divider} />
+                    </li>
+                )}
+                {regularCategories.map((category) => (
+                    <li className={styles.container} key={category.id}>
+                        <CategoryButton className={styles.categoryButton} category={category} />
+                    </li>
+                ))}
+            </>
+        );
+    }
+
+    return (
+        <>
+            <li className={navigationItemClassName}>
+                <Dropdown
+                    label={
+                        <FormattedMessage locale={localeCode} for={translations.categories.title} />
+                    }
+                    buttonClassName={navigationItemButtonClassName}
+                    withMobileDisplay
+                >
```

Comment:
Should there be a bit more top padding on the dropdown?

Right now, the dropdown button is "touching" the first row of featured categories, so might be a bit difficult to hit?

Currently in production there's more space between the button and the categories themselves.

Here:
![Screenshot 2024-03-20 at 13 00 46](https://github.com/prezly/theme-nextjs-bea/assets/4209081/57dcf768-ff14-4bc8-84bd-728478293693)

Prod:
![Screenshot 2024-03-20 at 13 25 04](https://github.com/prezly/theme-nextjs-bea/assets/4209081/94244fc0-0cfa-4ee7-8171-4009b56d0ea5)

Reply:
I guess maybe it might not be such a big problem if we add the placeholder category image.